dex/msgjson/types.go

```diff
@@ -1080,19 +1259,41 @@ type FeeAsset struct {
 	Amt   uint64 `json:"amount"`
 }
 
+// BondAsset describes an asset for which fidelity bonds are supported.
+type BondAsset struct {
+	Version uint16 `json:"version"` // latest version supported
+	ID      uint32 `json:"id"`
+	Confs   uint32 `json:"confs"`
+	Amt     uint64 `json:"amount"` // to be implied by bond version?
```

Comment:
Food for thought, but given variations in exchange rate, I think this has to be a variable.

Reply:
Note the entire solution, but in the right direction: https://github.com/decred/dcrdex/pull/1819/commits/e80d4a7ded182e6b63cdce5148ebccd7fee34c41

Still would need some kind of overlap handling like you said to avoid rejecting bonds that were confirming at the time of the Amt change.